src/web_interface/templates/show_analysis.html

```diff
@@ -217,6 +217,49 @@ <h2>Analysis for <span class="text-primary">{{ firmware.get_hid(root_uid=root_ui
                     </td>
                 </tr>
             {% endfor %}
+            <tr>
+                <td class="active result clickable" data-toggle="modal" data-target="#add_single_file_analysis_modal"><span class="glyphicon glyphicon-play-circle"></span> Run additional analysis</td>
+                <div id="add_single_file_analysis_modal" class="modal fade" role="dialog">
+                    <div class="modal-dialog">
+                        <div class="modal-content">
+
+                            <div class="modal-header">
+                                <button type="button" class="close" data-dismiss="modal">&times;</button>
+                                <h4 class="modal-title">Add analysis to file</h4>
+                            </div>
+
+                            <div class="modal-body">
+                                <form class="form-horizontal" action="" method=post enctype=multipart/form-data>
+                                    <p>Add new analysis</p>
+                                    {% for system in available_plugins[0] | sort %}
+                                        <div id="{{ system }}" class="{% if loop.index0 >= 0 %}col-xs-offset-3 {% endif %}col-xs-9">
+                                            <label class="checkbox-inline" data-toggle="tooltip" title="{{ analysis_plugin_dict[system][0] | safe }}">
+                                                <input type=checkbox name="analysis_systems" value="{{ system }}" unchecked>{{ system | replace_underscore }}<br />
+                                            </label>
+                                        </div>
+                                    {% endfor %}
+                                    <p>Update analysis</p>
+                                    {% for system in available_plugins[1] | sort %}
+                                        <div id="{{ system }}" class="{% if loop.index0 >= 0 %}col-xs-offset-3 {% endif %}col-xs-9">
+                                            <label class="checkbox-inline" data-toggle="tooltip" title="{{ analysis_plugin_dict[system][0] | safe }}">
+                                                <input type=checkbox name="analysis_systems" value="{{ system }}" unchecked>{{ system | replace_underscore }}<br />
+                                            </label>
+                                        </div>
+                                    {% endfor %}
+                                    <div style="padding: 15px">
+                                        <button type="submit" id="add_single_file_analysis" value=submit>Add</button>
```

Comment:
`class="btn btn-default"` is "missing" (has a different style than the "cancel" button)
maybe the cancel button should have the class `btn-danger` or `btn-warning` to get a visible distinction

Reply:
imho is would look better if the submit and cancel buttons were side by side (but this is debatable)